app/src/main/java/org/mifos/selfserviceapp/data/FundTransfer/FundTransferTemplateResponse.java

```diff
@@ -0,0 +1,30 @@
+package org.mifos.selfserviceapp.data.FundTransfer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Vishwajeet
+ * @since 23/8/16.
+ */
+
+public class FundTransferTemplateResponse {
+    private List<FundTransferTemplate> fromAccountOptions = new ArrayList<>();
+    private List<FundTransferTemplate> toAccountOptions = new ArrayList<>();
+
+    public List<FundTransferTemplate> getFromAccountOptions() {
+        return fromAccountOptions;
+    }
+
+    public void setFromAccountOptions(List<FundTransferTemplate> fromAccountOptions) {
```

Comment:
yeah. it's not something with this PR. In general, you expose methods that are intended to be used. In this case, set method is not intended to be used.

This is something you can keep in mind in general. Everything you expose a public method, think about whether it's needed..

You may choose to fix it in this PR, or tackle them generally in future PRs. Up to you.